migrations/20200917163300-create-events.js

```diff
@@ -0,0 +1,22 @@
+exports.up = (db, callback) => db.createTable('event', {
+  id: { type: 'int', primaryKey: true, autoIncrement: true },
+  type: 'string',
+  label: 'string',
+  model: 'string',
+  modelId: 'int',
+  createdAt: { type: 'date', notNull: true },
+  body: { type: 'jsonb', defaultValue: '{}' },
+})
+  .then(() => Promise.all([
+    db.runSql('CREATE INDEX IF NOT EXISTS "event_type_idx" ON event USING btree ("type");'),
+    db.runSql('CREATE INDEX IF NOT EXISTS "label" ON event USING btree ("label");'),
+    db.runSql('CREATE INDEX IF NOT EXISTS "model" ON event USING btree ("model");'),
+    db.runSql('CREATE INDEX IF NOT EXISTS "modelId" ON event USING btree ("modelId");'),
```

Comment:
I imagine that when querying by `model` we'll generally be querying on both `model` and `modelId`, maybe it would make more sense to have a compound index on both columns instead.